client/src/components/Logs.tsx

```diff
@@ -353,38 +367,24 @@ const EnvironmentLogs = ({ name, annotations }: EnvironmentLogsProps) => {
  * @param {function} toggleLogs - toggle logs visibility and fetch logs on show
  * @param {object} logs - log object from redux store enhanced with `show` property
  * @param {string} name - server name
- * @param {object} annotations - list of annotations
+ * @param {ReactNode | string} title - modal title
  */
 interface EnvironmentLogsPresentProps {
-  annotations: Record<string, unknown>;
+  title: ReactNode | string;
```

Comment:
```suggestion
  title: ReactNode;
```

`string` is already part of `ReactNode`.

Reply:
@leafty  fixed in the latest commit